styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -62,4 +72,46 @@ public FlyteExecution createExecution(final String name, final FlyteExecConf fly
       throw new CreateExecutionException(flyteExecConf, e);
     }
   }
+
+  public void poll(final String project, final String domain, final String name, RunState runState) {
```

Comment:
I think that we should create or own version of [WorkflowExecutionIdentifier](https://github.com/lyft/flyteidl/blob/fc787dc41985215ab025fbd55de14764fd7efa03/protos/flyteidl/core/identifier.proto#L39) and use it here instead of passing the three values independently

Reply:
I created the FlyteExecution for that, i can reuse it

Reply:
👍